src/main/java/controller/BlackJackGame.java

```diff
@@ -0,0 +1,170 @@
+            OutputView.printCardsHeldByPlayer(playerName, playerCards);
+        }
+    }
+
+    private static void askWantMoreCardToAllPlayers(Players players) {
+        players.getPlayers().forEach(BlackJackGame::askWantMoreCardToPlayer);
+    }
+
+    private static void askWantMoreCardToPlayer(Player player) {
+        if (player.isBust() || player.isBlackJack()) {
+            return;
+        }
+        chooseAnswer(player);
+    }
+
+    private static void chooseAnswer(Player player) {
+        String playerName = player.getPlayerName().getName();
+        OutputView.printDoYouWantOneMoreCard(playerName);
+        Answer answer = new Answer(InputView.inputAnswer());
+        if (answer.isYes()) {
+            distributeCardToPlayer(player);
+        }
+    }
+
+    private static void distributeCardToPlayer(Player player) {
+        Deck.distributeCard(player);
+        String playerName = player.getPlayerName().getName();
+        String playerCardsInfo = convertUserToCards(player);
+        OutputView.printCardsHeldByPlayer(playerName, playerCardsInfo);
+        askWantMoreCardToPlayer(player);
+    }
+
+    private static void distributeCardToDealer(Dealer dealer) {
+        if (dealer.hasSmallNumberLessThanRuleNumber()) {
+            OutputView.printDealerGetOneMoreCard();
+            Deck.distributeCard(dealer);
+            distributeCardToDealer(dealer);
+        }
+    }
+
+    private static void showDealerAndPlayersResult(Players players, Dealer dealer) {
+        showDealerResult(dealer);
+        showPlayersResult(players);
+    }
+
+    private static void showDealerResult(Dealer dealer) {
+        String dealerCardsInfo = convertUserToCards(dealer);
+        int dealerTotalCardNumber = dealer.calculateTotalCardNumber();
+        OutputView.printDealerResult(dealerCardsInfo, dealerTotalCardNumber);
+    }
+
+    private static void showPlayersResult(Players players) {
+        for (Player player : players.getPlayers()) {
+            String playerName = player.getPlayerName().getName();
+            String playerCardsInfo = convertUserToCards(player);
+            int totalCardNumber = player.calculateTotalCardNumber();
+            OutputView.printPlayerResult(playerName, playerCardsInfo, totalCardNumber);
+        }
+    }
+
+    private static String convertUserToCards(User user) {
+        return user.getCards().stream()
+                .map(card -> card.getType().getTypeName() + card.getSymbol().getSymbolName())
+                .collect(joining(SEPARATOR));
+    }
+
+    private static void showDealerAndPlayersProfit(Players players, Dealer dealer) {
+        PlayerProfitStatistics playerProfitStatistics = new PlayerProfitStatistics(players.producePlayersProfitStatistics(dealer));
+        showDealerProfit(dealer, playerProfitStatistics);
+        showPlayersProfit(players, playerProfitStatistics);
+    }
+
+    private static void showDealerProfit(Dealer dealer, PlayerProfitStatistics playerProfitStatistics) {
```

Comment:
```suggestion
    private static void showDealerProfit(PlayerProfitStatistics playerProfitStatistics) {
```
dealer를 파라미터로 받아와서  ``showDealerProfit`` 내부에서 사용하지 않는 것 같아요🤔

Reply:
놓쳤던 부분이네요! 꼼꼼한 피드백 감사합니다 :)